src/LowercaseSentenceTokenizerTest.java

```diff
@@ -19,6 +19,25 @@ void testTokenizeWithNoCapitalizationOrPeriod() {
     /*
      * Write your test here!
      */
+    @Test
+    void LowercaseSentenceTokenizer() {
+
+        //Arrange
+        LowercaseSentenceTokenizer tokenizer = new LowercaseSentenceTokenizer();
+
+        
+        //Action
+        Scanner scanner = new Scanner("hello     hi hi hi    hello hello");
+        List<String> tokens = tokenizer.tokenize(scanner);
+        // Debug Console
+       
+
+        //Assertion
+        // I am not understanding why my actual and my asserts are different?
+        assertEquals(List.of("hello", "hi" , "hi" , "hi" , "hello" , "hello"), tokens);
+       
+        
+    }
```

Comment:
Nice test! It looks like maybe there's some outdated comments you can remove, and you don't need so many empty between your logical lines of code.